cmd/argoexec/commands/root.go

```diff
@@ -90,6 +90,9 @@ func initExecutor() *executor.WorkflowExecutor {
 	clientset, err := kubernetes.NewForConfig(config)
 	checkErr(err)
 
+	restClient := clientset.RESTClient()
```

Comment:
@alexec We have a winner here! We can instantiate the RESTClient from the exising clientset without introducing new direct dependency (`k8s.io/apiextensions-apiserver` was already in `go.sum` partially due to this reason). The rest of the code remains the same.

This is also what `kubectl` uses under the hood (should've looked into its implementation in the first place!).